packages/web-components/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -0,0 +1,236 @@
+  /* --- Default styles --- */
+
+  :host {
+    max-width: fit-content;
+  }
+
+  :host .heading {
+    height: 44px;
+    display: grid;
+    position: relative;
+    vertical-align: middle;
+    padding-left: ${spacingHorizontalMNudge};
+    padding-right: ${spacingHorizontalM};
+    border-radius: ${borderRadiusMedium};
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    font-weight: ${fontWeightRegular};
+    line-height: ${lineHeightBase300};
+    grid-template-columns: 28px 28px auto 28px;
+  }
+
+  :host .heading .heading-content {
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+  :host .button {
+    box-sizing: border-box;
+    appearance: none;
+    border: none;
+    outline: none;
+    text-align: left;
+    cursor: pointer;
+    font-family: inherit;
+    height: 44px;
+    color: ${colorNeutralForeground1};
+    background: ${colorTransparentBackground};
+    line-height: ${lineHeightBase300};
+    height: auto;
+    padding: 0;
+    font-size: inherit;
+    grid-column: auto / span 2;
+    grid-row: 1;
+  }
+
+  :host .button::before {
+    content: '';
+    position: absolute;
+    inset: 0px;
+    cursor: pointer;
+    border-radius: ${borderRadiusSmall};
+  }
+
+  :host .icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    pointer-events: none;
+    position: relative;
+    height: 100%;
+    padding-right: ${spacingHorizontalS};
+    grid-column: 1 / span 1;
+    grid-row: 1;
+  }
+
+  :host .region {
+    display: none;
+    margin-left: ${spacingHorizontalM};
+    margin-right: ${spacingHorizontalM};
+  }
+
+  :host .heading ::slotted(*) {
+    display: flex;
+    align-self: center;
+  }
+
+  :host slot[name='collapsed-icon'],
+  :host slot[name='expanded-icon'] {
+    display: flex;
+    align-items: center;
+  }
+
+  :host slot[name='expanded-icon'] {
+    display: none;
+  }
+
+  :host .heading ::slotted(span[slot='start']),
+  :host .heading ::slotted(span[slot='end']) {
+    justify-content: center;
+    align-items: center;
+    padding-right: ${spacingHorizontalS};
+    grid-column: 2 / span 1;
+    grid-row: 1 / span 1;
+  }
+
+  :host button:focus-visible::after {
+    content: '';
+    position: absolute;
+    inset: 0px;
+    cursor: pointer;
+    border-radius: ${borderRadiusSmall};
+    outline: none;
+    border: 2px solid ${colorStrokeFocus1};
+    box-shadow: inset 0 0 0 1px ${colorStrokeFocus2};
+  }
+
+  /* --- Disabled attr styles --- */
+
+  :host([disabled]) .button {
+    color: ${colorNeutralForegroundDisabled};
+  }
+  :host([disabled]) svg {
+    filter: invert(89%) sepia(0%) saturate(569%) hue-rotate(155deg) brightness(88%) contrast(87%);
+  }
+
+  /* --- Expanded attr styles --- */
+
+  :host([expanded]) .region {
+    display: block;
+  }
+
+  :host([expanded]) slot[name='collapsed-icon'] {
+    display: none;
+  }
+
+  :host([expanded]) slot[name='expanded-icon'] {
+    display: flex;
+  }
```

Comment:
Is there a reason we're adding a display property to the `slot` here? By design, the default display property of the slot is `display: contents` which doesn't produce a box itself but defers to the "slotted".

From MDN:
```
These elements don't produce a specific box by themselves. They are replaced by their pseudo-box and their child boxes. Please note that the CSS Display Level 3 spec defines how the contents value should affect "unusual elements" — elements that aren't rendered purely by CSS box concepts such as replaced elements. See [Appendix B: Effects of display: contents on Unusual Elements](https://drafts.csswg.org/css-display/#unbox) for more details.
```
If you're hitting something unusual which is causing us to specifically style a slot, we should look at an alternative. Setting the display property of a slot element itself should be considered an anti-pattern for our scenarios.

Reply:
Friendly ping on this specific scenario - is there a reason we are targeting the slot's display property? Was something not working as expected?